packages/xo-web/src/common/xo/copy-vms-modal/index.js

```diff
@@ -75,8 +77,14 @@ class CopyVmsModalBody extends BaseComponent {
       copyMode: 'fullCopy',
       namePattern: '{name}_COPY',
     })
+    this.isCurrentPlanHigherThanStarter = getXoaPlan().value > STARTER.value
   }
 
+  getSrPredicate = createSelector(
+    () => this.props.resolvedVms,
+    vms => sr => every(vms, { $poolId: sr.$pool })
```

Comment:
Also check that the SR is writable with [`isSrWritable`](https://github.com/vatesfr/xen-orchestra/blob/18762dc624cbb0d77d606d98619814fc67a442ab/packages/xo-web/src/common/xo/index.js#L65) (which is the [default predicate](https://github.com/vatesfr/xen-orchestra/blob/18762dc624cbb0d77d606d98619814fc67a442ab/packages/xo-web/src/common/select-objects.js#L429)).